### DISCUSSION TITLE
What the hell is this, not linebreaking when no spaces

### DISCUSSION THREAD

**Discussion by HotoRas** [community/community#13209] [Other Feature Feedback, Questions, & Ideas] [answered]
Is this a git's original bug or Feed's?
![Screenshot](https://user-images.githubusercontent.com/34373595/159401743-4b64e981-8741-4ecb-9a6d-404441b5c52c.png)
Desc: A person setted his desc as a bunch of 'a's, and this ended for me to see the left side of the site which is not supposed to be shown.

well I wonder just if this is fixable lol

**Comment by jamieshark** [comment, marked as answer]
Thanks! We've filed this as a bug and will have a fix out soon.
